backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/UserService.cs

```diff
@@ -21,5 +27,22 @@ public async Task<UserDto> CreateUser(UserRegisterDto userDto)
 
             return _mapper.Map<UserDto>(userEntity);
         }
+
+        public async Task<LessonDto> EnrollUserToLesson(long lessonId)
+        {
+            var user = _context.Users
+                .Single(p => p.Id == _firebaseAuthService.UserId);
```

Comment:
use Single is not a good idea

Reply:
Pls explain? Your arguments?

Reply:
I've found options like `find`, `first`. Will try with `find`.

Reply:
Please do not resolve a conversation without an issue fixing
Yes, we in this way you will have an ability to throw your custom exception with custom message that will be more readable for the end user

And yes, it would be even better use SingleOrDefault instead of FirstOrDefault

Reply:
Implemented with `SingleOrDefault()`